covid-19/active-reserve-personnel/js/lines.js

```diff
@@ -0,0 +1,75 @@
+Highcharts.setOptions({
+  lang: {
+      numericSymbols: null 
+  }
+});
+
+$(function() {
+  $('#hcContainer').highcharts({
+    // Load Data in from Google Sheets
+    data: {
+      googleSpreadsheetKey: '1OzJwZ8wTlflcKsb_yKfmGchhkAzUkQ-fexGwY-c-sAk',
+      googleSpreadsheetWorksheet: 1
+    },
+    // General Chart Options
+    chart: {
+      zoomType: 'x',
+      type: 'line'
+    },
+    // Colors
+    colors: [
+      '#83373E',
+      '#365F5A'
+      
+    ],
+    // Chart Title and Subtitle
+    title: {
+      text: "Active and Reserve Personnel in COVID-19 Response"
+    },
+    subtitle: {
+      text: "Click and drag to zoom in"
+    },
+    // Credits
+    credits: {
+      enabled: true,
+      href: false,
+      text: "CSIS Defense360 | Source: Department of Defense"
+    },
+    // Chart Legend
+    legend: {
+      align: 'center',
+      verticalAlign: 'bottom',
+      layout: 'horizontal'
+    },
+    // Y Axis
+    yAxis: {
+      title: {
+        text: "Number of Personnel"
+      }
+    },
+    xAxis: {
+      dateTimeLabelFormats: {
+        day: '%b %e',
+        week: '%b %e'

```

Comment:
Let's switch the date format to dd-mmm to be consistent with the existing charts